app/routes/api/v1/warehouse/orders/by-customer-id/$customerId.ts

```diff
@@ -0,0 +1,24 @@
+import { Order } from "@prisma/client";
+import { LoaderFunction } from "@remix-run/node";
+import { badRequest, notFoundRequest } from "~/helpers/app-helpers.server";
+import { findCustomerOrders } from "~/models/order/order.server";
+
+type LoaderData = Order[];
+
+export const loader: LoaderFunction = async ({
+  params,
+}): Promise<LoaderData> => {
+  const customerId = params.customerId;
+
+  if (!customerId) {
+    throw badRequest();
+  }
+
+  const orders = await findCustomerOrders(customerId);
+
+  if (!orders || !orders.length) {
+    throw notFoundRequest();
+  }
+
+  return orders;
+};
```

Comment:
I would refactor the API path in something like `/api/v1/customers/$customerId/orders`

Reply:
ok, yeah, that sounds like a better solution